bedboss/bedqc/bedqc.py

```diff
@@ -0,0 +1,105 @@
+import logging
+import os
+import tempfile
+import pypiper
+import subprocess
+
+from bedboss.const import MAX_FILE_SIZE, MAX_REGION_SIZE, MIN_REGION_WIDTH
+from bedboss.exceptions import QualityException
+
+
+_LOGGER = logging.getLogger("bedboss")
+
+
+def bedqc(
+    bedfile: str,
+    outfolder: str,
+    max_file_size: int = MAX_FILE_SIZE,
+    max_region_size: int = MAX_REGION_SIZE,
+    min_region_width: int = MIN_REGION_WIDTH,
+    pm: pypiper.PipelineManager = None,
+    **kwargs,
+) -> bool:
+    """
+    Main pipeline function
+    :param bedfile: path to the bed file
+    :param outfolder: path to the folder where to store information about pipeline and logs
+    :param max_file_size: maximum file size
+    :param max_region_size: maximum region size
```

Comment:
These are not helpful docstrings, they just repeat what the self-documenting variable name is saying. Can you be a little more helpful in the docstring explanation?